src/_data/LiveData.js

```diff
@@ -28,6 +28,13 @@ const handlers = {
 
 export const api = new LiveApi({ language: 'EN' });
 
+const subscribeToSelectedSymbol = () => {
+    loadedStorePromise.then(st => {
```

Comment:
Yes, I note this in the `Refactor` todo, do we want to do it now?
I forgot why I did not refactor it directly, I believe because a few places need to be change